app/controllers/MessageController.js

```diff
@@ -0,0 +1,31 @@
+Message = require('../models/Message');
+Chat = require('../models/Chat');
+
+exports.createChat = (req, res) => {
+    if(!req.body.text) {
```

Comment:
@adilet-reactjs I think we don't need text here
Because we're not sending a message yet ))